wrapper/telemetry-wrapper.js

```diff
@@ -353,14 +352,14 @@ window.TelemetryWrapper.go = function (params, element) {
     var percentileLabel = ' - medians'; // i18n?
     var valuesArePercent = false;
     if (kind == 'enumerated' || kind == 'boolean' || kind == 'flag') {
-      const BUCKET_INDEX_FOR_ENUMERATED = 0;
+      const bucketIndex = params.evoBucketIndex > 0 ? params.evoBucketIndex : 0;
       if (kind == 'boolean') {
         yLabel = desc + ' % FALSE'; // TODO: i18n
       } else {
-        yLabel = desc + ' - bucket ' + BUCKET_INDEX_FOR_ENUMERATED; // i18n?
+        yLabel = desc + ' - bucket ' + bucketIndex; // i18n?
       }
       valueses = evolutions.map(evo =>
-        evo.map(hist => 100 * hist.values[BUCKET_INDEX_FOR_ENUMERATED] / hist.count
+        evo.map(hist => 100 * hist.values[bucketIndex] / hist.count
```

Comment:
This has a possibility of throwing an exception if is `bucketIndex >= hist.values.length`